scipy/optimize/_shgo.py

```diff
@@ -305,58 +303,58 @@ def shgo(func, bounds, args=(), constraints=None, n=100, iters=1, callback=None,
     ...
     >>> bounds = [(-512, 512), (-512, 512)]
 
-    `shgo` has two built-in low discrepancy sampling sequences. First, we will
-    input 30 initial sampling points of the Sobol sequence:
+    `shgo` has built-in low discrepancy sampling sequences. First, we will
+    input 64 initial sampling points of the Sobol' sequence:
 
-    >>> result = shgo(eggholder, bounds, n=30, sampling_method='sobol')
+    >>> result = shgo(eggholder, bounds, n=64, sampling_method='sobol')
     >>> result.x, result.fun
-    (array([ 512.        ,  404.23180542]), -959.64066272085051)
+    (array([512.        , 404.23180824]), -959.6406627208397)
 
     `shgo` also has a return for any other local minima that was found, these
     can be called using:
 
     >>> result.xl
-    array([[ 512.        ,  404.23180542],
-           [ 283.07593402, -487.12566542],
+    array([[ 512.        ,  404.23180824],
+           [ 283.0759062 , -487.12565635],
            [-294.66820039, -462.01964031],
-           [-105.87688985,  423.15324143],
-           [-242.97923629,  274.38032063],
+           [-105.87688911,  423.15323845],
+           [-242.97926   ,  274.38030925],
            [-506.25823477,    6.3131022 ],
-           [-408.71981195, -156.10117154],
-           [ 150.23210485,  301.31378508],
-           [  91.00922754, -391.28375925],
-           [ 202.8966344 , -269.38042147],
-           [ 361.66625957, -106.96490692],
-           [-219.40615102, -244.06022436],
-           [ 151.59603137, -100.61082677]])
+           [-408.71980731, -156.10116949],
+           [ 150.23207937,  301.31376595],
+           [  91.00920901, -391.283763  ],
+           [ 202.89662724, -269.38043241],
+           [ 361.66623976, -106.96493868],
+           [-219.40612786, -244.06020508]])
 
     >>> result.funl
     array([-959.64066272, -718.16745962, -704.80659592, -565.99778097,
            -559.78685655, -557.36868733, -507.87385942, -493.9605115 ,
-           -426.48799655, -421.15571437, -419.31194957, -410.98477763,
-           -202.53912972])
+           -426.48799655, -421.15571437, -419.31194957, -410.98477763])
 
     These results are useful in applications where there are many global minima
     and the values of other global minima are desired or where the local minima
     can provide insight into the system (for example morphologies
-    in physical chemistry [5]_).
+    in physical chemistry [4]_).
 
     If we want to find a larger number of local minima, we can increase the
     number of sampling points or the number of iterations. We'll increase the
-    number of sampling points to 60 and the number of iterations from the
-    default of 1 to 5. This gives us 60 x 5 = 300 initial sampling points.
+    number of sampling points to 64 and the number of iterations from the
+    default of 1 to 5. Using ``simplicial`` this would have gives us
+    64 x 3 = 192 initial sampling points.
```

Comment:
64 * 5 is 320.

Reply:
Or are you increasing the number of iterations from 1 to 3?

Reply:
Was 3 indeed.